assets/javascripts/kitten/components/tables/list-table/components/col.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import COLORS from '../../../../constants/colors-config'
+import { ScreenConfig } from '../../../../constants/screen-config'
+import {
+  CONTAINER_MAX_WIDTH,
+  CONTAINER_PADDING_THIN,
+  CONTAINER_PADDING,
+} from '../../../../constants/grid-config'
+import { pxToRem } from '../../../../helpers/utils/typography'
+import { Context } from './context'
+
+const StyledItem = styled.li`
+  flex-grow: 0;
+  flex-shrink: 0;
+
+  &:not(:first-child) {
+    padding-left: ${pxToRem(CONTAINER_PADDING_THIN)};
+  }
+
+  @media screen and (min-width: ${ScreenConfig.M.min}px) {
```

Comment:
Quel est le rôle d'ajouter `screen` par rapport à:
`@media screen and (min-width: ${ScreenConfig.M.min}px)`

Reply:
Pour que ces styles ne s'appliquent pas à l'impression—dans le cas présent c'est vrai, je sais pas si c'est bien utile, mais dans beaucoup de cas en responsive ça l'est, c'est pour ça que je le fais un peu automatiquement :)